### PR TITLE
Application.css: increase padding

### PR DESCRIPTION
**Pull request by danrabbit**
Since that's our standard window padding

### CODE REVIEW

**Comment by cassidyjames** [comment]
Hrm, this feels really weird to me. Maybe because the padding is not even all the way around? But it feels like the sides are way too padded inside the terminal view itself.

**Comment by danrabbit** [comment]
Yeah I can't see a way to pad the top and bottom without it affecting the scroll area which looks weird.

Maybe see if it grows on you. I could settle for 6px or 9px too. But the current padding is just so little

File: data/Application.css [hunk 1]
```diff
@@ -26,8 +26,7 @@
 }
 
 vte-terminal {
-    padding-left: 4px;
-    padding-right: 4px;
+    padding: 0 12px;
```
Comment (lainsce) [inline data/Application.css:29]: How about this? Makes it look more uniform and less lopsided. :smile: 

```suggestion
    padding: 12px 12px 0 12px;
```

**Comment by cassidyjames** [comment]
Currently experimenting with `12px 6px` plus an undershoot.

Screenshot from 2020-08-10 12 13 10

File: data/Application.css (hunk 1 above)
Comment (cassidyjames) [reply data/Application.css:29]: So this still has the weird scrolling problem. I think just applying to the sides for now is fine unless/until we can sort out the undershoot.

Comment (cassidyjames) [reply data/Application.css:29]: @danrabbit I think 0 and 12px feels too lopsided, so unless/until we can sort out some sort of undershoot or other way to handle that, how about `0 9px`? Still a notable improvement over master.

```suggestion
    padding: 0 9px;
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Let's try `0 9px` for now?

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**